v2/kafka-to-gcs/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToGCS.java

```diff
@@ -0,0 +1,264 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link KafkaToGCS} pipeline reads message from Kafka topic(s) and stores to Google Cloud
+ * Storage bucket in user specified format. The sink data can be stored in a Text, Avro or a Parquet
+ * File Format.
+ *
+ * <p><b>Pipeline Requirements</b>
+ *
+ * <ul>
+ *   <li>Kafka Bootstrap Server(s).
+ *   <li>Kafka Topic(s) exists.
+ *   <li>Google Cloud Storage output bucket exists.
+ * </ul>
+ *
+ * <p><b>Example Usage</b>
+ *
+ * <pre>
+ * # Set the pipeline vars
+ * PROJECT=my-project
+ * BUCKET_NAME=my-bucket
+ *
+ * # Set containerization vars
+ * IMAGE_NAME=my-image-name
+ * TARGET_GCR_IMAGE=gcr.io/${PROJECT}/${IMAGE_NAME}
+ * BASE_CONTAINER_IMAGE=my-base-container-image
+ * BASE_CONTAINER_IMAGE_VERSION=my-base-container-image-version
+ * APP_ROOT=/path/to/app-root
+ * COMMAND_SPEC=/path/to/command-spec
+ *
+ * # Build and upload image
+ * mvn clean package \
+ * -Dimage=${TARGET_GCR_IMAGE} \
+ * -Dbase-container-image=${BASE_CONTAINER_IMAGE} \
+ * -Dbase-container-image.version=${BASE_CONTAINER_IMAGE_VERSION} \
+ * -Dapp-root=${APP_ROOT} \
+ * -Dcommand-spec=${COMMAND_SPEC}
+ *
+ * # Create an image spec in GCS that contains the path to the image
+ * {
+ *    "docker_template_spec": {
+ *       "docker_image": $TARGET_GCR_IMAGE
+ *     }
+ *  }
+ *
+ * # Execute template:
+ * API_ROOT_URL="https://dataflow.googleapis.com"
+ * TEMPLATES_LAUNCH_API="${API_ROOT_URL}/v1b3/projects/${PROJECT}/templates:launch"
+ * JOB_NAME="kafka-to-gcs-`date +%Y%m%d-%H%M%S-%N`"
+ *
+ * time curl -X POST -H "Content-Type: application/json"     \
+ *     -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+ *     "${TEMPLATES_LAUNCH_API}"`
+ *     `"?validateOnly=false"`
+ *     `"&dynamicTemplate.gcsPath=${BUCKET_NAME}/path/to/image-spec"`
+ *     `"&dynamicTemplate.stagingLocation=${BUCKET_NAME}/staging" \
+ *     -d '
+ *      {
+ *       "jobName":"'$JOB_NAME'",
+ *       "parameters": {
+ *           "bootstrapServers":"broker_1:9092,broker_2:9092",
+ *           "inputTopics":"topic1,topic2",
+ *           "outputDirectory":"'$BUCKET_NAME/path/to/output-location'",
+ *           "outputFileFormat":"text",
+ *           "outputFilenamePrefix":"output",
+ *           "windowDuration":"5m",
+ *           "numShards":"5"
+ *        }
+ *       }
+ *      '
+ * </pre>
+ */
+public class KafkaToGCS implements WriteToGCS {
+
+  /* Logger for class. */
+  private static final Logger LOG = LoggerFactory.getLogger(KafkaToGCS.class);
+
+  /**
+   * The {@link KafkaToGCSOptions} class provides the custom execution options passed by the
+   * executor at the command-line.
+   */
+  public interface KafkaToGCSOptions extends PipelineOptions {
+    @Description("Kafka Bootstrap Servers. ")
+    @Required
+    String getBootstrapServers();
+
+    void setBootstrapServers(String bootstrapServers);
+
+    @Description("Kafka topic to read the input from. ")

```

Comment:
Comma separated list of input topics to read from?

Reply:
Even this does accept a list.